generator/integration_tests/golden/golden_idempotency_policy_test.cc

```diff
@@ -119,8 +119,8 @@ TEST_F(GoldenIdempotencyPolicyTest, ListBackupOperations) {
   EXPECT_EQ(policy_->ListBackupOperations(request), Idempotency::kIdempotent);
 }
 
-}  // namespace
-}  // namespace golden_internal
-}  // namespace GOOGLE_CLOUD_CPP_NS
-}  // namespace cloud
-}  // namespace google
+} // namespace
```

Comment:
I think two spaces before the EOL comment like it was before is preferred in the style guide.